client/src/components/QandA/QuestionsList/QuestionCard.jsx

```diff
@@ -1,32 +1,166 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import QuestionBody from './QuestionCard/QuestionBody.jsx';
 import Answers from './Answers.jsx';
 import QuestionDetails from './QuestionCard/QuestionDetails.jsx';
 
-const QuestionCard = ({ answers, asker, body, date, helpful, reported, ID }) => {
+class QuestionCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answerCounter: 0,
+      showMoreBtn: false,
+      showCollapseBtn: false
+    };
+    this.onSeeMoreClick = this.onSeeMoreClick.bind(this);
+    this.onCollapseClick = this.onCollapseClick.bind(this);
+  }
+
+  componentDidMount() {
+    this.initialSetter();
```

Comment:
where are you defining this function? i see it called here and on line 22 but i dont see it defined anywhere

Reply:
This confused me just slightly right now too. Lol. It _is_ defined on Line 22. The commented out line above Line 22 makes it a little confusing. I have a commented out line on L53. I'll remove that.